### ISSUE TITLE
Compare ML Algorithms in Drug Discovery

### ISSUE THREAD

**Issue by cathyghotbi** [cathyghotbi/drug_discovery_ML#18]
**Acceptance Criteria:**

    investigate ML algorithms in drug discovery
    collect the information in wiki

**Description:**
Learn about/ Implement:

-  Logistic Regression
-  Random Forest
-  Gradient Boosting (XGBoost or LightGBM)
-  SVM
-  kNN

For each:
-  Cross-validation
-  Hyperparameter tuning (GridSearchCV)
-  Report mean ± std ROC-AUC

**Comment by cathyghotbi** [comment]
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Random-Forest
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Logistic-Regression
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Decision-Tree

https://github.com/cathyghotbi/drug_discovery_ML/wiki/Classification-and-Regression
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Supervised-vs-Unsupervised-Learning
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Overfitting
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Underfitting
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Regularization
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Linear-Relationship
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Nonlinear-Relationship
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Feature-Engineering
https://github.com/cathyghotbi/drug_discovery_ML/wiki/Data-Leakage
